3.evaluate_model/get_LOIO_probabilities_all_featurespaces_no_retraining.ipynb

```diff
@@ -0,0 +1,1258 @@
+{
```

Comment:
<p>Very well documented, easy to follow</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/phenotypic_profiling_model/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Great! It is maybe worth noting that this is mostly Roshan's code ported over to this new file.